Code/Yahoo.Yui.Compressor.Build.MsBuild/CompressorTaskEngine.cs

```diff
@@ -399,6 +400,17 @@ private bool SaveCompressedText(StringBuilder compressedText)
         {
             // Note: compressedText CAN be null or empty, so no check.
 
+            var folderNamePattern = @"(.*)\\(.*)\.(.*)$";
+            if (Regex.IsMatch(OutputFile, folderNamePattern))

```

Comment:
Can you please explain what this regex is trying to do. Please give some scenario examples. (eg. input & output).